webserver/views/api/v1/dataset_eval.py

```diff
@@ -2,71 +2,72 @@
-                           "description": null,
-                           "id": "141",
-                           "name": "Class2",
-                           "recordings": [
-                               "d08ab44b-94c8-482b-a67f-a683a30fbe5a",
-                               "2618cb1d-8699-49df-93f7-a8afea6c914f"
-                           ]
-                       },
-                       {
-                           "description": null,
-                           "id": "142",
-                           "name": "Class1",
-                           "recordings": [
-                               "5251c17c-c161-4e73-8b1c-4231e8e39095",
-                               "c0dccd50-f9dc-476c-b1f1-84f00adeab51"
-                           ]
-                       }
-                   ],
-                   "created": "Mon, 02 May 2016 16:41:08 GMT",
-                   "description": "",
-                   "id": "5375e0ff-a6d0-44a3-bee1-05d46fbe6bd5",
-                   "last_edited": "Mon, 02 May 2016 16:41:08 GMT",
-                   "name": "test4",
-                   "public": true
-               },
-               "dataset_id": "5375e0ff-a6d0-44a3-bee1-05d46fbe6bd5",
-               "eval_location": "local",
-               "id": "7804abe5-58be-4c9c-a787-22b91d031489",
-               "options": {
-                   "filter_type": null,
-                   "normalize": false
-               },
-               "result": null,
-               "snapshot_id": "2d51df50-6b71-410e-bf9a-7e877fc9c6c0",
-               "status": "pending",
-               "status_msg": null,
-               "testing_snapshot": null,
-               "training_snapshot": null,
-               "updated": "Tue, 07 Jun 2016 22:12:32 GMT"
-           },
-          "success": true
-    }
+           'username': 'Test'
+           'jobs': [
+               {
+                   'dataset_name': 'test_dataset',
+                   'created': 'Wed, 08 Jun 2016 21:41:58 GMT',
+                   'job_id': '0fc553b6-f3a8-4e24-af39-56ac02276ed9'
+               }
+           ]
+       }
+
+    :reqheader Content-Type: *application/json*
+    :<json string username:  MusicBrainz username of the user.
+    :<json array jobs:       Jobs which match the query.
+    """
+
+    location = request.args.get("location")
+    status = request.args.get("status")
+    # TODO: This endpoint can be used in the future to get any type of job,
+    #       but for now we only use it for pending/remote.
+    if location != db.dataset_eval.EVAL_REMOTE or status != db.dataset_eval.STATUS_PENDING:
+        raise exceptions.APIBadRequest("parameter location must be 'remote' and status 'pending'")
+
+    jobs = db.dataset_eval.get_remote_pending_jobs_for_user(current_user.id)
+    return jsonify(
+        username=current_user.musicbrainz_id,
+        jobs=jobs
+    )
+
+@bp_dataset_eval.route("/bulk-recordings/<recording_ids>", methods=["GET"])
+def bulk_recordings(recording_ids):
```

Comment:
I would suggest to reuse existing endpoint by providing an option to specify list of IDs as an argument. I don't like the trend of creating more and more endpoints that provide multiple ways to get the same data. There should be one place to get a particular entity.

Think about how people are going to use the API, not just a client that you are building. It's hard to change things in the API once they are there.